.gitignore

```diff
@@ -18,7 +18,6 @@ tmp/*
 vgp
 bin/*
 .envrc
-athens
 yarn-error.log
```

Comment:
Let us remove it as well, since we have removed buffalo altogether, there is no reason that we should have yarn-log as well.